src/routes/resources/+page.svelte

```diff
@@ -2,8 +2,10 @@
   import type { PageData } from "./$types";
   import { onMount } from "svelte";
   import { DEFAULT_DISPLAY_LIMIT } from "$lib/constants";
-  import type { Tag, FilterOption, FilterLogic } from "$lib/interfaces";
+  import type { Tag, FilterOption, FilterLogic, Resource } from "$lib/interfaces";
   import { setIntersection } from "$lib/utils";
+  import Collapsible from "$lib/components/Collapsible.svelte";
+  import Search from "lib//components/Search.svelte";
```

Comment:
`lib//` vs `$lib/`
and not sure Collapsible is in this file?

Reply:
Collapsible has been removed from this file, so there is no need to import it here.
lib// vs $lib/ appears to work either way, but changing it to $lib/ to keep it consistent with the other imports.